R/download_zenodo.R

```diff
@@ -14,30 +15,32 @@
 #' @importFrom jsonlite fromJSON
 #' @importFrom tools md5sum
 #' @importFrom utils tail
-#'
+#' @importFrom assertthat
+#' assert_that
+#' is.string
 #'
 #' @export
 #' @family download
 #'
 #' @examples
 #' \dontrun{
-#' # Single zip file deposition
-#' download_zenodo(".", "10.5281/zenodo.1283345")
-#' # Multiple files deposition
-#' download_zenodo(".", "10.5281/zenodo.1172801")
-#' # Single pdf file deposition
-#' download_zenodo(".", "10.5281/zenodo.168478")
+#' # Single zip file download
+#' download_zenodo(doi = "10.5281/zenodo.1283345")
+#' # Multiple files download
+#' download_zenodo(doi = "10.5281/zenodo.1172801")
+#' # Single pdf file download
+#' download_zenodo(doi = "10.5281/zenodo.168478")
 #' }
-download_zenodo <- function(path, doi) {
-  if (missing(path)) {
-    stop("Please provide a path to which the data need to be downloaded")
-  }
+download_zenodo <- function(doi,
+                            path = ".") {
   if (missing(doi)) {
```

Comment:
this is somewhat redundant. The function will fail is `doi` is missing. The required format of `doi` is defined in the documentation.
IMHO it is more important to check the correct format when `doi` is given.

Reply:
Resolved in 465ef00